enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/NeoShallowEmbeddedInteraction.java

```diff
@@ -40,13 +40,13 @@
 
 class NeoShallowEmbeddedInteraction implements NeoInteractionLevel<EnterpriseAuthSubject>
 {
-    private GraphDatabaseAPI db;
+    private GraphDatabaseFacade db;
```

Comment:
This can be GraphDatabaseAPI, which has both execute (from GraphDatabaseService) and beginTransaction